src/instrumentations/index.ts

```diff
@@ -43,12 +43,24 @@ const supportedInstrumentations: [string, string][] = [
   ['opentelemetry-instrumentation-mongoose', 'MongooseInstrumentation'],
 ];
 
-export function getInstrumentations(): InstrumentationOption[] {
```

Comment:
Are these changes neccesary given `configureLogInjection` function also checks if log injection is enabled and skips configuration if not? We could remove these checks and always return all the instrumentations. If a user has installed one of these packages, it is likely they want log injection but even if they don't want it and it is installed, `configureLogInjection` will later just not configure these instrumentations.

Reply:
Good point, changed it to always load them.

And added a comment to the tests when checking for the amount of instrumentations loaded by default